kernel/config.arm64

```diff
@@ -1,6 +1,6 @@
 #
```

Comment:
This file doesn't belong to FS change

Reply:
kernel changes in this branch are now in #81